src/App.js

```diff
@@ -11,6 +13,7 @@ import {
 
 const HomeScreen = ({navigation}) => (
   <View style={{flex: 1, alignItems: 'center', justifyContent: 'center'}}>
+    <ProgressBar></ProgressBar>
```

Comment:
this will make it appear only on home screen. we probs want it to be visible across the whole app.